utils_entrypoint.py

```diff
@@ -0,0 +1,33 @@
+
+
+def load_default_args(match_config):
```

Comment:
A common pattern for this kind of defaults config is to creatre a defaults dict and then overwrite it with use config.

e.g.

```python

DEFAULTS = {
        "closest_match_variable":  None,
        "date_exclusion_variables": None,
        "min_matches_per_case": 0,
        "replace_match_index_date_with_case": None,
        "output_suffix": "",
        "indicator_variable_name": "case",
        "output_path": "tests/test_output",
        "input_path": "tests/test_data",
        "drop_cases_from_matches": False,
}

def load_config(match_config):
    cfg = DEFAULTS.copy()
    cfg.update(match_config)
    return cfg

```